firebase-abt/firebase-abt.gradle

```diff
@@ -87,4 +86,5 @@ dependencies {
     testImplementation 'io.grpc:grpc-testing:1.12.0'
     testImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 
+    compileOnly "com.google.protobuf:protobuf-javalite:$javaliteVersion"
```

Comment:
AFAIK firebase-abt cannot be used stand alone. Can it?

Reply:
ABT can't be used alone (since you have to run the ABT test on something else) but we should update the comment if we don't move the proto definition into FIAM.